gamepanel/member.php

```diff
@@ -442,18 +450,24 @@ function memberUserDetail()
 
 		if( $this->Game instanceof panelGameBoard
 			&& $this->status == 'Playing' && $this->Game->phase != 'Finished' )
-		{
+		{	
+			if(!$this->isMissedTurnsHidden()){
+				if($this->Game->excusedMissedTurns > 0)
+					$buf .= ' - '.l_t('NMRs: %s of %s','<span class="excusedNMRs">'.$this->excusedMissedTurns.'</span>','<span class="excusedNMRs">'.$this->Game->excusedMissedTurns.'</span>');
+				
+				if ( $this->missedPhases == 2 )

```

Comment:
I'm not sure I'm understanding what this section of the code is going to be doing. We should never get to a counter of missedPhases == 2 right? At that point they should have been kicked out of the game if they were out of excused missed turns?

Reply:
The missedPhases part is actually old code that I just left there for the time being to display missed phases as before. At first I was confused about the 2, as well, though the old code works in the following way: After a turn is processed all members missed phases counter is increased by 1 and is only reset to 0 if the member loads the board. So two missed phases in the counter actually mean, that the member was absent during the whole turn before.

Though, as we could have multiple misses in a row, this should be replaced in any case. I guess it would be easiest to keep this missed Phases counter and just replace the "==" by ">=" and increase the counter in any case at the end of the process method. Working instead with the actual database entries could be a headache here as the turn does not progress and we would have to work with the turnAsDate and all the complications of extended phases etc..